docs/_posts/murat-gunay/2022-01-18-bert_sequence_classifier_sentiment_da.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: model
+title: Sentiment Analysis of Danish texts
+author: John Snow Labs
+name: bert_sequence_classifier_sentiment
+date: 2022-01-18
+tags: [danish, sentiment, da, open_source]
+task: Sentiment Analysis
+language: da
+edition: Spark NLP 3.3.4
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model was imported from `Hugging Face` and it's been fine-tuned for Danish language, leveraging Danish Bert embeddings and `BertForSequenceClassification` for text classification purposes.
```

Comment:
I think sometimes we include the link to the model in the description, sometimes we don't. Let's include it always: https://huggingface.co/DaNLP/da-bert-tone-sentiment-polarity

And then, in data source, just the link to the data.

Reply:
@josejuanmartinez 
I was trying to be consistent with previously uploaded HF models by Maziyar. Maziyar (also me) has been uploaded tons of models providing hugging face page link in "dataset" section (which is more correct for me).
If we change this policy, we should change all previously uploaded models in models&hub. What do you suggest?

Reply:
@murat-gunay  there are a lot of models with the model in description as well. There has not been any fixed criteria. Please, let's add the model link to the description section and the dataset to the dataset link to the dataset section. Thanks.

Reply:
@josejuanmartinez what if we rename the `dataset` section to `reference` and we just put all the citations there? Like the original model, a dataset that was used, even the original GitHub repo if we have, we just list them all in there. This way we don't have to go back and fix maybe 4000+ and it will be correct with whatever it's inside?

Reply:
@maziyarpanahi makes perfect sense! How should we reach out to?

Reply:
@josejuanmartinez I think @pabla team will do this in 1 minute, they change and merge it.